src/Tempest/Database/Migrations/MigrationManager.php

```diff
@@ -27,21 +27,21 @@ public function up(): void
         try {
             $existingMigrations = Migration::all();
         } catch (PDOException) {
-            $this->executeUp(new CreateMigrationsTable());
+            $this->executeUp(new CreateMigrationsTable($this->databaseConfig->driver));
```

Comment:
at the moment passing the driver to the migration, as I mentioned above I'm looking if I can skip this